tests/test_scene_rendering/simple_scenes.py

```diff
@@ -6,3 +6,12 @@ def construct(self):
         square = Square()
         circle = Circle()
         self.play(Transform(square, circle))
+
+
+class SceneWithMultipleCalls(Scene):
+    def construct(self):
+        number = Integer(0)
+        self.add(number)
+        for i in range(10):
+            number.become(Integer(i))
+            self.play(Animation(number))
```

Comment:
```suggestion
            self.play(Animation(number))

```

Just added a newline at the end.